controllers/openstack_ansibleee_controller.go

```diff
@@ -175,6 +175,19 @@ func (r *OpenStackAnsibleEEReconciler) Reconcile(ctx context.Context, req ctrl.R
 			fmt.Sprintf("Creating a new Job: Job.Namespace %s Job.Name %s\n", job.Namespace, job.Name),
 			instance,
 		)
+
+		if len(instance.Spec.EnvConfigMapName) == 0 {
+			instance.Spec.EnvConfigMapName = "openstack-aee-default-env"
+		}
```

Comment:
I don't think the above "if" is needed since we are setting a default with kubebuilder. What do you think?

Reply:
kubebuilder default is applied when you create AEE from yaml, as we also create AEE from go code, we need it
(that's the reason behind https://github.com/openstack-k8s-operators/nova-operator/blob/b03835b621125c84f162b0c00697f5419818197b/api/v1beta1/novaexternalcompute_types.go#L140)

Reply:
ok, makes sense, thanks!